.drone/deploy/server/image/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
```

Comment:
Add a comment that this script exists only to translate `$PORT` to `$DRONE_SERVER_PORT` so that we can run it in Heroku?